src/hotspot/cpu/x86/sharedRuntime_x86.cpp

```diff
@@ -87,6 +87,7 @@ void SharedRuntime::inline_check_hashcode_from_object_header(MacroAssembler* mas
 #if defined(TARGET_COMPILER_gcc) && !defined(_WIN64)
 JRT_LEAF(jfloat, SharedRuntime::frem(jfloat x, jfloat y))
   jfloat retval;
+  if (UseAVX < 1 || !UseFMA) {
```

Comment:
SharedRuntime::frem is called for both 32bit and 64bit. 
The new stubs are only for 64 bit so something like below:
if (!is_LP64 || UseAVX < 1 || !UseFMA)